luigi/parameter.py

```diff
@@ -219,6 +219,8 @@ def parse(self, x):
         :param str x: the value to parse.
         :return: the parsed value.
         """
+        if not isinstance(x, basestring) or not isinstance(x, str):
```

Comment:
My mac is not playing nice installing python 3.5 and setting up a venv, but does anyone know if this line will falsely evaluate that a string (unicode or ascii) is not an instance of basestring for python 3+?

This does seem to work correctly for python 2.7.